demo/molecule/autosuggest/demo/index.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/prop-types, no-unused-vars, no-console */
+import React from 'react'
+
+import {withStateValue, withStateValueTags} from '@s-ui/hoc'
+
+import MoleculeAutosuggest, {
+  MoleculeAutosuggestDropdownListSizes,
+  withDynamicOptions
+} from '../../../../components/molecule/autosuggest/src'
+import MoleculeAutosuggestOption from '@s-ui/react-molecule-dropdown-option'
+
+import AutosuggestSingleWithAsyncOptions from './components/AutosuggestSingleFromAjax'
+
+import {IconClose} from './Icons'
+import {getAsyncCountriesFromQuery} from './services'
+import './index.scss'
+
+const MoleculeAutosuggestWithDynamicOptions = withDynamicOptions(
+  MoleculeAutosuggest,
+  MoleculeAutosuggestOption
+)(getAsyncCountriesFromQuery)
+
+const MoleculeAutosuggestWithState = withStateValue(
+  MoleculeAutosuggestWithDynamicOptions
+)
+
+const MoleculeAutosuggestWithStateTags = withStateValueTags(
+  MoleculeAutosuggestWithDynamicOptions
+)
+
+const BASE_CLASS_DEMO = 'DemoMoleculeAutosuggest'
+const CLASS_DEMO_SECTION = `${BASE_CLASS_DEMO}-section`
+
+const Demo = () => (
+  <div className={BASE_CLASS_DEMO}>
+    {/*
```

Comment:
Well, i thought of letting this so the final developer can uncomment this in case they want to see this _custom_ use of `MoleculeAutosuggest` in action. Also thought, the _purpose_ of this component is too technical to be part of the final demo